io.c

```diff
@@ -8251,25 +8314,10 @@ rb_io_print(int argc, const VALUE *argv, VALUE out)
 
 /*
  *  call-seq:
- *     print(obj, ...)    -> nil
+ *    print(*objects)    -> nil
  *
- *  Prints each object in turn to <code>$stdout</code>. If the output
- *  field separator (<code>$,</code>) is not +nil+, its
- *  contents will appear between each field. If the output record
- *  separator (<code>$\\</code>) is not +nil+, it will be
- *  appended to the output. If no arguments are given, prints
- *  <code>$_</code>. Objects that aren't strings will be converted by
- *  calling their <code>to_s</code> method.
- *
- *     print "cat", [1,2,3], 99, "\n"
- *     $, = ", "
- *     $\ = "\n"
- *     print "cat", [1,2,3], 99
- *
- *  <em>produces:</em>
- *
- *     cat12399
- *     cat, 1, 2, 3, 99
+ *  Equivalent to <tt>$stdout.print(*objects)</tt>.
+ *  See IO#print.
```

Comment:
Maybe leave here some small example, and only then say "See IO#print for more detailed examples"?
`print` is more frequently used method than `IO#print`, and I personally hate when the method I want to use sends me to "some other place" for explanations. (Also, for novice it can be confusing: trying to print something, they'll be navigated to `IO#print` and copy examples from there, and then will for several months or years be sure that the "right" way is to do `$stdout.print(...)` instead of just `print(...)`)

Reply:
Since Kernel#print is the most-used, I've fleshed it out entirely, with no 'see'.  I think the stuff should also remain at IO#print.

[When I was a young documentor, I _always_ used a macro processor, so that very similar passages could share source.  Where's m4 when you need it?]